harness/determined/keras/_tf_keras_trial.py

```diff
@@ -112,6 +124,10 @@ def on_train_batch_end(self, _: int, logs: Any = None) -> None:
 
         self.tf_keras_trial_controller.run()
 
+        if self.model.stop_training and version.parse(tf.__version__) >= version.parse("2.0.0"):
+            # Starting with TF 2.2, `model.stop_training` is only checked at the end of epochs.
```

Comment:
question: it's a little odd to me that the comment is talking about 2.2 but the check is for 2.0. Was this a mistake?

Reply:
Oh yeah this is wrong, thanks for catching it.